clang/unittests/Tooling/DependencyScanning/DependencyScannerTest.cpp

```diff
@@ -235,9 +235,12 @@ TEST(DependencyScanner, ScanDepsWithFS) {
   VFS->addFile(TestPath, 0,
                llvm::MemoryBuffer::getMemBuffer("#include \"header.h\"\n"));
 
-  DependencyScanningService Service(ScanningMode::DependencyDirectivesScan,
-                                    ScanningOutputFormat::Make, CASOptions(),
-                                    nullptr, nullptr, nullptr);
+  DependencyScanningService Service(
+      ScanningMode::DependencyDirectivesScan, ScanningOutputFormat::Make,
+      CASOptions(), nullptr, nullptr, nullptr, ScanningOptimizations::Default,
+      /*EagerLoadModules=*/false,
+      /*TraceVFS=*/false, llvm::sys::toTimeT(std::chrono::system_clock::now()),
+      true);
```

Comment:
Can you add `/*CacheNegativeStats=*/` label here other places the service gets constructed?

Reply:
Why is it set to true here? I'd think we'd want to match the default behavior of the libclang API.